app/client/templates/layouts/master_layout/master_layout.html

```diff
@@ -9,7 +9,7 @@
     (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
     (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
     m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
-    })(window,document,'script','//www.google-analytics.com/analytics.js','ga');
+    })(window,document,'script','https://www.google-analytics.com/analytics.js','ga');
```

Comment:
I'd be happy with stuffing this whole thing inside an `if (window.location.host !== "localhost")` block or something like that, to make the intent explicit rather than something that behaves as desired but prints a loud warning to the console on every pageload.